src/cmd/config.go

```diff
@@ -27,7 +29,7 @@ var configSchemaCmd = &cobra.Command{
 	Long:  "Print the jsonschema for configuration file",
 	Run: func(cmd *cobra.Command, args []string) {
 		schema := jsonschema.Reflect(&common.Config{})
-		jsonBytes, err := json.MarshalIndent(schema, "", "  ")
+		jsonBytes, err := json.MarshalIndent(schema, "", "    ")
```

Comment:
why is this necessary?

Reply:
@rocktavious I was fixing indentation in YAML and JSON files and along the way I wanted JSON to display with 4 spaces. CLI also uses 4 spaces and while it is not a standard this is the most common.